Reply/app/src/main/java/com/example/reply/data/Account.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.reply.data
+
+import androidx.annotation.DrawableRes
+
+/**
+ * An object which represents an account which can belong to a user. A single user can have
+ * multiple accounts.
+ */
+data class Account(
+    val id: Long,
+    val uid: Long,
+    val firstName: String,
+    val lastName: String,
+    val email: String,
+    val altEmail: String,
+    @DrawableRes val avatar: Int,
```

Comment:
[suggestion] referencing a DrawableRes for a data class that generally could be stored to disk (or come from a server), is not a great idea as the resources aren't stable values (when you build a new APK it might change the value). Have you considered using a url here instead?

Reply:
Hey I am using the local drawable's for this sample and local dummy data is created so it will always have reference. Do you mean url as in upload storage and url ?

Reply:
As long as its not stored to disk and then the reference read from storage on a new app run it should work, but my suggestion I guess is that this isn't really a good practise since this kind of data normally comes from a server, I guess its fine for this instance though 🤷🏻